srcCpp/Infrastructure_micro.h

```diff
@@ -13,6 +13,7 @@
 #include "osapi/osapi_time.h"
 #include "rti_me_cpp.hxx"
 #include "PerftestTransport.h"
+#include "CustomCRC.h"
```

Comment:
We should protect this, right? Else, it will not work with versions prior to 3.0.3.X, will it?